spec/factories/category.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :category do |f|
+    f.parent_id {rand(0..10)}
```

Comment:
fake tào lao, chém chừ, chỗ ni sao lại random dc cậu, nó là 1 giá trị phải tồn tại trong DB mà 
```suggestion
    f.parent_id {Category.first&.id || ""} # nghĩa là lấy thèn category đầu tiên là cha của nó, hoặc nó chính là cha 
```